src/api/URLs.ts

```diff
@@ -9,7 +9,11 @@ class URLs {
         private static resultsPerPage: number = 20;
 
     private static baseUrl(path?: string): URL {
-        return new URL(API_BASE_URL + "/api" + path);
+        return new URL(API_BASE_URL + path);
+    }
+
+    private static APIBaseUrl(path?: string): URL {
```

Comment:
If path is undefined, then I think the string "undefined" would be concatenated onto "/api"